### PR TITLE
Declare DD_VERSION in prerun.sh

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] DX/Feature

## Description
Following the example provided by [DataDog's doc](https://docs.datadoghq.com/agent/basic_agent_usage/heroku/#prerun-script), I've added assignment of DD_VERSION for deployment tracking purpose

## Related Tickets & Documents
n/a
## QA Instructions, Screenshots, Recordings
n/a

## Added tests?
- [x] No, and this is why: n/a

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._
- [x] This change does not need to be communicated, and this is why not: n/a

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

File: datadog/prerun.sh [hunk 1]
```diff
@@ -8,4 +8,9 @@ if [ "$DYNOTYPE" == "run" ]; then
   # more info here: https://docs.honeycomb.io/getting-data-in/ruby/beeline/#using-env-variables-to-control-framework-integrations
   export HONEYCOMB_DISABLE_AUTOCONFIGURE="true"
   export HONEYCOMB_INTEGRATIONS=rails,rake,active_support
+
+  # Set app version based on HEROKU_SLUG_COMMIT
+  if [ -n "$HEROKU_SLUG_COMMIT" ]; then
+    DD_VERSION=$HEROKU_SLUG_COMMIT
```
Comment (jgaskins) [inline datadog/prerun.sh:14]: Should this be `export`ed like the ones above?

```suggestion
    export DD_VERSION=$HEROKU_SLUG_COMMIT
```

**Review by jgaskins — COMMENTED** [review]
Awesome find! Just a quick thing to check and then I would love to see this go out! :100:

File: datadog/prerun.sh (hunk 1 above)
Comment (maestromac) [reply datadog/prerun.sh:14]: I think DD's env vars do not require to be exported. `DISABLE_DATADOG_AGENT` wasn't exported in this file.

Comment (jgaskins) [reply datadog/prerun.sh:14]: Oh interesting. Well, let's give it a shot and see how it goes!

**Review by jgaskins — APPROVED** [review]

**Review by Zhao-Andy — APPROVED** [review]

**Pull request merged**